roadie-tech-docs/docs/configuration/tech-radar/index.md

```diff
@@ -0,0 +1,111 @@
+
+## Introduction
```

Comment:
Could we put all this duplicate content into a different GitHub repo? I'd like to have separation between the content which is the source of truth and the content which is just a result of the build process.

Reply:
Done